src/deps/picohttpparser.zig

```diff
@@ -1,21 +1,66 @@
-pub usingnamespace @import("std").zig.c_builtins;
+const std = @import("std");
 
+/// contains name and value of a header (name == `null` if is a continuing line of
+/// a multiline header)
 pub const struct_phr_header = extern struct {
     name: [*c]const u8,
     name_len: usize,
     value: [*c]const u8,
     value_len: usize,
 };
+/// returns number of bytes consumed if successful, -2 if request is partial, -1
+/// if failed
 pub extern fn phr_parse_request(buf: [*c]const u8, len: usize, method: [*c][*c]const u8, method_len: [*c]usize, path: [*c][*c]const u8, path_len: [*c]usize, minor_version: [*c]c_int, headers: [*c]struct_phr_header, num_headers: [*c]usize, last_len: usize) c_int;
+/// returns number of bytes consumed if successful, -2 if request is partial, -1
+/// if failed
 pub extern fn phr_parse_response(_buf: [*c]const u8, len: usize, minor_version: [*c]c_int, status: [*c]c_int, msg: [*c][*c]const u8, msg_len: [*c]usize, headers: [*c]struct_phr_header, num_headers: [*c]usize, last_len: usize) c_int;
+/// returns number of bytes consumed if successful, -2 if request is partial, -1
+/// if failed
 pub extern fn phr_parse_headers(buf: [*c]const u8, len: usize, headers: [*c]struct_phr_header, num_headers: [*c]usize, last_len: usize) c_int;
-pub const struct_phr_chunked_decoder = extern struct {
+pub const ChunkedDecoder = extern struct {
```

Comment:
This was named to match the struct declaration in the C source code.

Reply:
is `picohttp.ChunkedDecoder` not clear enough?